docs/sources/k6/next/javascript-api/k6-browser/metricmessage.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 'MetricMessage'
+slug: 'metricmessage'
+description: 'Browser module: MetricMessage Class'
+weight: 03
+---
+
+# MetricMessage
+
+`MetricMessage` objects are dispatched by page via registering a handler with [page.on('metric')](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-browser/page/on). For each metric that it measured and emitted for the page, k6 browser delivers it to the registered handlers.
+
+## tag
+
+The Metric Message exposes a single method, `tag`. It will match the given `matches` with the current metric's url and name tags. When a match is found it will use `name` to replace the existing url and name tag values.
+
+Doing this will help group metrics with disparate url and name tags, which are in fact referencing the same resource, so that a correlation can be found over time and preventing high cardinality issues in the backend database that is storing the metrics.
```

Comment:
I would remove the backend part from this text as it's related to k6 internals.

Reply:
Sure, updated in 98464b654fd8c72ea439cd061636d4d7489f0a1f